pkg/search/handler.go

```diff
@@ -808,14 +809,14 @@ func (h *Handler) EdgesTo(req *EdgesRequest) (*EdgesResponse, error) {
 		if found {
 			ei = &EdgeItem{
 				From:     edge.From,
-				FromType: "permanode",
+				FromType: string(schema.PermanodeCamliType),
```

Comment:
Why not change the EdgeItem.FromType type?

Reply:
Done